.coveragerc

```diff
@@ -446,6 +446,11 @@ omit =
     homeassistant/components/mychevy/*
     homeassistant/components/mycroft/*
     homeassistant/components/mycroft/notify.py
+    homeassistant/components/myio_valet_mega/__init__.py
```

Comment:
By adding all files to .coveragerc you signal your code is not covered by testing. 

Furthermore I understand from our conversation in discord that your dev container do not work. Please secure the code you submit work locally for you.

Also please write sone test cases.